.rubocop.yml

```diff
@@ -592,27 +592,18 @@ Rails/WhereRange: # new in 2.25
 
 ################## RSpec #################################
 
-RSpec/BeEmpty: # new in 2.20
-  Enabled: true
-
-RSpec/BeEq: # new in 2.9.0
```

Comment:
Seems we probably want this disabled? Lots of local correctable offences with it enabled

RSpec/BeNil may also have a config we want to select - there's some correctable offences there, like:

```
spec/helpers/application_helper_spec.rb:4:52: C: [Corrected] RSpec/BeNil: Prefer be_nil over be(nil).
      expect(helper.send(:breakable_text, nil)).to be(nil)
                                                   ^^^^^^^
```

Reply:
Huh, I don't recall those showing up when I checked, weird. Seems reasonable to restore that if so, yeah.

Reply:
Ah, this second one came from an `eq(nil)` becoming `be(nil)` becoming `be_nil`

But yeah, the underlying RSpec/BeEq seems better to keep disabled so we don't have like 300 correctable offences churned